graphistry/dgl_utils.py

```diff
@@ -225,8 +205,8 @@ def dgl_lazy_init(self, train_split: float = 0.8, device: str = "cpu"):
         """
 
         if not self.dgl_initialized:
-            lazy_dgl_import_has_dependency()
-            lazy_torch_import_has_dependency()
+            deps.dgl
+            deps.torch
```

Comment:
What is the point of these calls, we need to throw an exn if missing?

If so, can we access the original exn and rethrow?